From 5b85473c19e098e41cd76f9506a7a86a47222df5 Mon Sep 17 00:00:00 2001
From: Matthew Joyce <matthew.joyce@refeyn.com>
Date: Mon, 22 May 2023 15:43:35 +0100
Subject: [PATCH] Remove internal dependencies and C extension from setup

---
 setup.py | 28 ++--------------------------
 1 file changed, 2 insertions(+), 26 deletions(-)

diff --git a/setup.py b/setup.py
index 8e116b235..cc5e7a33d 100644
--- a/setup.py
+++ b/setup.py
@@ -11,7 +11,6 @@ import sys
 from pathlib import Path
 
 import setuptools
-from setuptools_rust import RustExtension  # type: ignore[import]
 
 sys.path.insert(0, str(Path(__file__).parent / "internal_dependencies" / "dev_utils"))
 from dev_utils.deploy.makeUtils import (
@@ -67,40 +66,17 @@ SETUP_KWARGS = {
     "author": "Refeyn Ltd",
     "author_email": "max.hantke@refeyn.com",
     "license": "Proprietary and confidential",
-    "packages": compile_package_list(PATH_PACKAGE)
-    + compile_package_list(INTERNAL_DEPENDENCIES_PATH / "iscat" / "iscat")
-    + compile_package_list(INTERNAL_DEPENDENCIES_PATH / "iscat_utils" / "iscat_utils"),
+    "packages": compile_package_list(PATH_PACKAGE),
     "package_dir": {
-        PACKAGE_NAME: str(PATH_PACKAGE),
-        **{n: str(p) for n, p in PACKAGE_DEPENDENCIES.items()},
+        PACKAGE_NAME: str(PATH_PACKAGE)
     },
     "include_package_data": True,
     "entry_points": {"console_scripts": ["lw8 = lw8.lw8:main"]},
-    "ext_modules": [
-        setuptools.Extension(
-            name="lw8.devices.stage.mcs2_c",
-            sources=[str(PATH_PACKAGE / "devices" / "stage" / "mcs2.cpp")],
-            include_dirs=[
-                str(PATH_MCS2 / "include"),
-                # Numpy include dirs are added later by build_ext.finalize_options in build_ext_numpy_included
-            ],
-            library_dirs=[str(PATH_MCS2 / "lib64")],
-            libraries=["SmarActCTL"],
-        )
-    ],
     "build_ext": build_ext_numpy_included,
 }
 
 
 if __name__ == "__main__":
-    if "bdist_wheel" in sys.argv:
-        SETUP_KWARGS["rust_extensions"] = [
-            RustExtension(
-                "iscat._accelerated",
-                "internal_dependencies/iscat/crates/iscat_accelerated/Cargo.toml",
-                debug=False,
-            )
-        ]
     if "sdist" in sys.argv:
         generate_combined_manifest(
             PATH_PACKAGE.parent / "MANIFEST.in",
