From 2b34ce5f0ac3436188d7e48e2e4a687baeba4674 Mon Sep 17 00:00:00 2001
From: Matthew Joyce <matthew.joyce@refeyn.com>
Date: Thu, 19 Sep 2024 16:10:59 +0100
Subject: [PATCH] Disable cachegen to fix lines not showing up

---
 QuickGraphLib/GraphItems/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/QuickGraphLib/GraphItems/CMakeLists.txt b/QuickGraphLib/GraphItems/CMakeLists.txt
index 725233d..c939c62 100644
--- a/QuickGraphLib/GraphItems/CMakeLists.txt
+++ b/QuickGraphLib/GraphItems/CMakeLists.txt
@@ -24,6 +24,7 @@ qt_add_qml_module(QuickGraphLibGraphItems
     SOURCES helpers/HistogramHelper.cpp helpers/HistogramHelper.hpp helpers/GridHelper.hpp helpers/GridHelper.cpp
     PLUGIN_TARGET QuickGraphLibGraphItems
     DEPENDENCIES QtQuick QuickGraphLib
+    NO_CACHEGEN
 )
 
 target_compile_definitions(
